app-frontend/src/app/components/sceneDetail/sceneDetail.html

```diff
@@ -0,0 +1,13 @@
+<div class="content">
+  <img ng-if="$ctrl.scene.thumbnails[0]"
+       ng-attr-src="{{$ctrl.scene.thumbnails[0].url}}"
+       class="rounded-img detail-img">
+  <img ng-if="!$ctrl.scene.thumbnails.length"
+       src="http://placehold.it/275x275"
+       class="rounded-img detail-img">
+  <h5 class="color-dark">Image metadata</h5>
+  <dl class="meta-list">
+    <dt ng-repeat-start="(metaKey, metaVal) in $ctrl.scene.sceneMetadata">{{metaKey}}</dt>
```

Comment:
Two things here -- and maybe we leave it for now and figure out a design/code solution later:

 1. Should we try and convert everything to title case? There's going to be some limit to what we can do for the generic metadata, but the things that come from the database are going to be camelCase. So maybe we convert things from our database and the extra info (stored as `jsonb` in the database) we just leave.
 2. Some metadata values are _really_ long. Should we truncate and use a tooltip (but what if someone wants to copy something)?

@designmatty have any thoughts on these?